coriolis/osmorphing/base.py

```diff
@@ -427,6 +428,49 @@ def _configure_cloud_init_user_retention(self):
                     "Failed to reconfigure cloud-init to retain user "
                     "credentials. Error was: %s" % str(err)) from err
 
+    def _disable_installer_cloud_config(self):
+        installer_config_path = "/etc/cloud/cloud.cfg.d/99-installer.cfg"
+        if self._test_path(installer_config_path):
+            self._event_manager.progress_update(
+                "Disabling Ubuntu installer-generated cloud-config")
+            self._exec_cmd_chroot(
+                f"mv {installer_config_path} {installer_config_path}.bak")
+
+    def _ensure_cloud_init_enabled(self):
+        disabler_file = "/etc/cloud/cloud-init.disabled"
+        if self._test_path(disabler_file):
+            self._exec_cmd_chroot(f"rm {disabler_file}")
```

Comment:
This looks incomplete. There's a couple of more methods of disabling cloud-init, we should cover for those too.